core/conversion/converters/impl/unsqueeze.cpp

```diff
@@ -21,8 +21,13 @@ auto unsqueeze_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns().p
        auto dim = args[1].unwrapToInt();
 
        auto selfDim = util::toVec(self->getDimensions());
+       int64_t nbDims = selfDim.size();
+       TRTORCH_ASSERT(

```

Comment:
Use TRTORCH_CHECK instead of TRTORCH_ASSERT. Assert will through an error requesting a bug report